### PR TITLE
Docs update and custom image options update

### PR DESCRIPTION
**Pull request by frezbo**
Signed-off-by: Noel Georgi <git@frezbo.com>

* Fixes the doc update that was missed in the initial implementation.
* Pulls in the changes to update `description` and `distribution` from https://github.com/digitalocean/godo/pull/413

### CODE REVIEW

File: digitalocean/resource_digitalocean_image.go [hunk 1]
```diff
@@ -179,15 +179,28 @@ func resourceDigitalOceanCustomImageUpdate(ctx context.Context, d *schema.Resour
...
+		}
+		if d.HasChange("distribution") {
+			imageDistribution := d.Get("distribution").(string)
+			imageUpdateRequest.Distribution = imageDistribution
+		}
+
+		if d.HasChange("description") {
+			imageDescription := d.Get("description").(string)
+			imageUpdateRequest.Description = imageDescription
+		}
```
Comment (andrewsomething) [inline digitalocean/resource_digitalocean_image.go:197]:
```suggestion
		imageUpdateRequest := &godo.ImageUpdateRequest{
			Name:         imageName,
			Distribution: d.Get("distribution").(string),
			Description:  d.Get("description").(string),
		}

```

We can simplify this a bit. PUT is idempotent, so no reason not to build the full request.

**Review by andrewsomething — COMMENTED** [review]
Thanks for picking this up! I had meant to add these here after doing so in godo, but it seems to have slipped my mind.

It would be great to add an additional step to this test exercising the update method:

https://github.com/digitalocean/terraform-provider-digitalocean/blob/374e7a8e4f1f80bc841a36457d07dcf618c0de3d/digitalocean/resource_digitalocean_image_test.go#L13

File: digitalocean/resource_digitalocean_image.go (hunk 1 above)
Comment (frezbo) [reply digitalocean/resource_digitalocean_image.go:197]: I didn't noticed it was idempotent, was trying to change only fields that changed. I'll make this change

**Comment by frezbo** [comment]
@andrewsomething I'll add a test for `update` in a bit. Finishing off something.

**Review by andrewsomething — APPROVED** [review]

**Comment by andrewsomething** [comment]
I've gone ahead and written a test for the update method to unblock this: https://github.com/digitalocean/terraform-provider-digitalocean/pull/538/commits/21c3341059f6604c79c2fae202619e70eb02571c

Thanks again for your contributions @frezbo!

**Comment by frezbo** [comment]
@andrewsomething I'm so sorry. I forgot about this. Thanks for taking it up :+1: :heart:

**Pull request merged**